tests/server/routes/members-only-test.js

```diff
@@ -7,7 +7,7 @@ var db = require('../../../server/db');
 
 var supertest = require('supertest');
 
-describe('Members Route', function () {
+xdescribe('Members Route', function () {
```

Comment:
Ok. Was it failing? It's not worth blocking the PR for, but in general, I'd only x-out an existing test if it's failing.